content/logs.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Logs
+order: 32
+description: How to use your logs
+---
+
+The logs should be your first stop when troubleshooting any Galaxy issue. When working with Galaxy support, an agent will usually begin by consulting your logs. If a clearly noted issue has been marked there, you can save time by reading the logs and iterating on your code until the error goes away.
+
+You can reach the logs dashboard by clicking on your app, then clicking the "Logs" tab.
+
+<img src="/images/logs_section.png" style="margin: 1em 0;"/>
+
+<h3 id="tabs">Tabs</h3>
+
+There are several logging tabs you can click, below the "Logs" tab itself. These are:
+
+- All: for the combined view.
+- App: for application-specific messages and errors.
+- Service: for messages generated during the build process.
+- Errors: exclusively for errors, which are flagged in red in other tab views.
+
+Breakdown by tab will be especially useful if you have multiple containers running, and are regularly deploying.
+
+<h4 id="all">All</h4>
+
+Every message and error will be shown here.
+
+If your containers are working correctly and have been up for a while, this may be the most convenient tab to check. If you have multiple containers and builds succeeding and failing, this view may be too crowded.
+
+<h4 id="app">App</h4>
+
+Any messages or errors thrown by your app will be shown here, assuming our image builder was able to build your app and successfully launch it.
+
+Check this to see what your app is doing during normal operations. You can always add lines to your code to print what your app is doing for added visibility.
+
+<h4 id="service">Service</h4>
+
+If your app isn't successfully deploying, this is the tab you'll want to check. Our image builder tries to build your app into a container; if it fails, the failure will be noted here.
+
+A <a href="https://github.com/meteor/meteor/issues/8136">common error</a> is that a particular version of a package wasn't able to be built by our image builder. If this happens to your app, the output will frequently include a line like this, after the package install line:
```

Comment:
Are you still seeing this one frequently?  Newest version of Meteor should fix it.